### ISSUE TITLE
Is the test set of ShareRobot open source?

### ISSUE THREAD

**Issue by yangyioryy** [FlagOpen/RoboBrain#28]
Hello,RoboBrain is a great project! I have a question while reading.

I saw a test set for ShareRobot in the paper. However, I only saw a training set in BAAI/ShareRobot. Is the test set data open source?

I would be very grateful if you could help me solve the problem

**Comment by Akila-Ayanthi** [comment]
Hi, where is the test set available?
